lib/Importer/CMakeLists.txt

```diff
@@ -5,17 +5,39 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 add_definitions(-DGOOGLE_PROTOBUF_NO_RTTI)
 
-PROTOBUF_GENERATE_CPP(PROTO_SRCS
-                      PROTO_HDRS
-                      caffe.proto
+PROTOBUF_GENERATE_CPP(CAFFE_SRCS CAFFE_HDRS caffe.proto)
+PROTOBUF_GENERATE_CPP(ONNX_SRCS
+                      ONNX_HDRS
                       ${GLOW_THIRDPARTY_DIR}/onnx/onnx/onnx.proto)
 
+# NB: We need to copy the *.pb.h files to appropriately-prefixed paths to
+# placate FB's internal build system.  That is, we need:
+#
+#   #include "glow/caffe.pb.h"
+#
+# instead of:
+#
+#   #include "caffe.pb.h"
+#
+# Please don't optimize away the "useless" copies!
+add_custom_command(
+  OUTPUT ${GLOW_BINARY_DIR}/glow/caffe.pb.h
+  COMMAND cp ${CAFFE_HDRS} ${GLOW_BINARY_DIR}/glow/caffe.pb.h
```

Comment:
It is better to use `${CMAKE_COMMAND} -E copy_if_different` to make this portable.  Particularly since there is already a PR to make this build/work on Windows.  That also has the benefit of not causing a rebuild all the time since the timestamps on the file shouldn't change unless the content changes.